components/homme/src/preqx/share/prim_advance_mod.F90

```diff
@@ -557,12 +559,65 @@ end subroutine noxsolve
 
     call t_stopf('prim_advance_exp')
 !pw call t_adj_detailf(-1)
-    end subroutine prim_advance_exp
+  end subroutine prim_advance_exp
 
 
+!ftype logic
+!should be called with dt_remap, on 'eulerian' levels, only before homme remap timestep
+  subroutine applyCAMforcing_ps(elem,hvcoord,dyn_timelev,tr_timelev,dt_remap,nets,nete)
+  use control_mod, only : ftype
+  implicit none
+  type (element_t),       intent(inout) :: elem(:)
+  real (kind=real_kind),  intent(in)    :: dt_remap
+  type (hvcoord_t),       intent(in)    :: hvcoord
+  integer,                intent(in)    :: dyn_timelev,tr_timelev,nets,nete
+
+  call t_startf("ApplyCAMForcing")
+  if (ftype==0) then
+    call applyCAMforcing_dynamics(elem,hvcoord,dyn_timelev,           dt_remap,nets,nete)
+    call applyCAMforcing_tracers (elem,hvcoord,dyn_timelev,tr_timelev,dt_remap,nets,nete)
```

Comment:
It changes nothing in this PR but order of applyCAMforcing_ calls will matter in theta model for standalone code. Since in standalone we want to mimic what the whole model does, we should move _tracers functions before _dyn function. This way standalone ftype=0 matches whole model ftype=2. Next pr for theta will do it.